### ISSUE TITLE
Add to default port allocations wiki page

### ISSUE THREAD

**Issue by mattbostock** [cloudflare/alertmanager2es#2]
It'd be good to add an entry to the [Prometheus wiki](https://github.com/prometheus/prometheus/wiki/Default-port-allocations) to make alertmanager2es easier to find.

The default port number we're using is 9097, though that doesn't strictly fit under 'Core components'. Perhaps @brian-brazil or @discordianfish could suggest the right place?

**Comment by brian-brazil** [comment]
Non-exporter stuff has been taking ports from the exporter list, so 9239 is next.
